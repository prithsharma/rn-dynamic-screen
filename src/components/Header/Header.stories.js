import React from 'react';
import { storiesOf } from '@storybook/react-native';

import Header from './index';

storiesOf('Card', module)
  .add('default', () => (
    <Header
      imgUri="http://res.cloudinary.com/onefc/image/upload/v1554878097/events/Bangkok-2019-05-10/upcoming"
    />
  ));
