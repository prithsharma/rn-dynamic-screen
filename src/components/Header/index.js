import React from 'react';
import {
  Image,
  StyleSheet,
  View,
} from 'react-native';


let styles;

export default function Header() {
  return (
    <View style={styles.container}>
      <Image
        style={styles.logo}
        source={require('./logo.png')}
      />
    </View>
  );
}

styles = StyleSheet.create({
  container: {
    width: '100%',
    height: 64,
    backgroundColor: 'black',
    alignItems: 'center',
    justifyContent: 'center',
  },
  logo: {
    maxWidth: 96,
    resizeMode: 'cover',
    height: '80%',
  },
});
