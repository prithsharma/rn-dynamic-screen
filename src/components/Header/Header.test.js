/**
 * @format
 */

import 'react-native';
import React from 'react';
import { shallow } from 'react-native-testing-library';
import Header from './index';

it('Header renders correctly', () => {
  expect(shallow(
    <Header />,
  ).output).toMatchSnapshot();
});
