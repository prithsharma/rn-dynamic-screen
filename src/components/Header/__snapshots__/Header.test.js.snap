// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "height": 64,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/components/Header/assets/logo.png",
      }
    }
    style={
      Object {
        "height": "80%",
        "maxWidth": 96,
        "resizeMode": "cover",
      }
    }
  />
</View>
`;
